src/bundle/Resources/public/js/CKEditor/custom-tags/custom-tag-editing.js

```diff
@@ -0,0 +1,135 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import { toWidget, toWidgetEditable } from '@ckeditor/ckeditor5-widget/src/utils';
+import Widget from '@ckeditor/ckeditor5-widget/src/widget';
+
+import CustomTagCommand from './custom-tag-command';
+
+class CustomTagEditing extends Plugin {
+    static get requires() {
+        return [Widget];
+    }
+
+    defineSchema() {
+        const schema = this.editor.model.schema;
+
+        schema.register('customTag', {
+            isObject: true,
+            allowWhere: '$block',
+            allowAttributes: ['customTagName', 'values'],
+        });
+
+        schema.register('customTagContent', {
+            isBlock: true,
+            allowIn: 'customTag',
+            allowContentOf: '$root',
+        });
+    }
+
+    defineConverters() {
+        const conversion = this.editor.conversion;
+
+        conversion.for('editingDowncast').elementToElement({
+            model: 'customTag',
+            view: (modelElement, { writer: downcastWriter }) => {
+                const container = downcastWriter.createContainerElement('div', {
+                    'data-ezelement': 'eztemplate',
```

Comment:
Do we still use `ez` prefix?

Reply:
In richtext for backward compatibility yes.